nilearn/glm/tests/test_thresholding.py

```diff
@@ -337,7 +337,11 @@ def test_threshold_stats_img_surface_with_mask(surf_img_1d, surf_mask_1d):
 @pytest.mark.parametrize("threshold", [3.0, 2.9, DEFAULT_Z_THRESHOLD])
 @pytest.mark.parametrize("height_control", [None, "bonferroni", "fdr", "fpr"])
 def test_deprecation_threshold(surf_img_1d, height_control, threshold):
-    """Check warning thrown when threshold==old threshold."""
+    """Check warning thrown when threshold==old threshold.
```

Comment:
Wondering whether we're not overdoing it. 
Shouldn't we simply let the use know that the default has changed when the function with changed default is called ?

Reply:
Yep, that's very thoughtful of you.